+nosnoc/Options.m

```diff
@@ -333,6 +333,8 @@
         %---------------------------------------------------------------------%
 
         use_numerical_clock_state(1,1) logical = false % logical: instead of sum of $h$ being used for equidistant control steps use a simple integrated state.
+
+        sigma_smoothing(1,1) double {mustBePositive} = 1e-7
```

Comment:
split already now integrator options into separate class?

Reply:
I agree fundamentally that this needs to happen. However, it is not clear to me how exactly this should be done.
In the sense that there is a strong overlap between `nosnoc.solver.Options` and the new `nosnoc.integrator.Options` in the case of using the FESD integrator.
My initial suggestion is to have the solver options be a field of the integrator options similar to the `casADi` paradigm.